packages/wonder-blocks-tooltip/components/tooltip.md

```diff
@@ -0,0 +1,85 @@
+### Text anchor & text tooltip
+
+```js
+const React = require("react");
+const {Tooltip} = require("@khanacademy/wonder-blocks-tooltip");
+
+<Tooltip content={"The tooltip text"}>
+    Some text
+</Tooltip>
+```
+
+### Complex anchor & text tooltip
+
+In this example, we're no longer forcing the anchor root to be focusable, since the text input can take focus.
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {Tooltip} = require("@khanacademy/wonder-blocks-tooltip");
+
+<Tooltip forceAnchorFocusivity={false} content={"The tooltip text"}>
+    <View>
+        Some text
+        <input />
+    </View>
+</Tooltip>
+```
+
+### Substring anchor in scrollable parent
+
+```js
+const React = require("react");
+const {View, Text} = require("@khanacademy/wonder-blocks-core");
+const {Tooltip} = require("@khanacademy/wonder-blocks-tooltip");
+
+<div>
+    <div style={{height: 100, overflow: "auto", border: "1px solid", margin: 10,}}>
+        <div style={{height: "200vh"}}>
```

Comment:
Please use `View` in the examples.

Reply:
These are all changing in later PRs as more components are introduced, so I'm going leave as-is here.